stable/cert-manager/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: cert-manager
-version: 0.2.10
+version: 0.2.11
```

Comment:
You're adding a new feature, so that warrant a minor version bump instead of patch level.